public/app/features/plugins/admin/hooks/usePluginInfo.tsx

```diff
@@ -26,10 +26,17 @@ export const usePluginInfo = (plugin?: CatalogPlugin): PageInfoItem[] => {
   }
 
   if (Boolean(version)) {
-    info.push({
-      label: 'Version',
-      value: version,
-    });
+    if (plugin.isManaged) {
+      info.push({
+        label: 'Version',
+        value: 'Managed by Grafana',
+      });
```

Comment:
I think this is perfectly fine to iterate with. I would still recommend bringing to a UX feedback session before we go beyond dev. 

A simple alternate would be 'Managed by Grafana' -> 'Latest'

I would almost expect something more like the signature shield where we could have the display value as Latest and then tooltip text of "The version of this plugin is managed by Grafana Labs"

Reply:
> I would almost expect something more like the signature shield where we could have the display value as Latest and then tooltip text of "The version of this plugin is managed by Grafana Labs"

:100: this is perfect. It's actually an advantage for us to manage the plugin :trophy:

Reply:
Would you folks prefer I roll with "Latest" as a first go in that case? I'll see if I can fiddle with some badge if you think that's worthwhile too. 

I would like to get something in place first and then take to UX as you suggested @sympatheticmoose 👍 There's feature toggles in place to prevent us from turning this on automatically anyway so we'll be controlling this manually on a per instance basis to test with first.

Reply:
IMO I would do "Managed by Grafana Labs" + badge > Latest.